sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -135,6 +144,15 @@ export interface CertificateOperationError {
     readonly message?: string;
 }
 
+// @public
+export interface CertificateOperationPollOperationStatePublic extends PollOperationStateWithPublicState<CertificateOperationPublicState, KeyVaultCertificateWithPolicy> {
```

Comment:
I'm a bit confused by why so many interfaces are needed to accomplish returning the state of the poller to the user. Also there are so many permutations of the words "Poller", "State" and "Public" that it's easy to loose track of what each type is for.

Reply:
These interfaces are so that the return types of the poller methods are simplified, since we're avoiding having to export the actual poller classes. I agree that this is getting confusing, but I don't know how to fix it.